src/pages/costDetails/detailsToolbar.tsx

```diff
@@ -121,7 +121,7 @@ export class DetailsToolbar extends React.Component<DetailsToolbarProps> {
             onClick={this.toggleCurrentSortDirection}
           />
         </Sort>
-        <Icon name="download" size="2x" />
+        {/* <Icon name="download" /> */}
```

Comment:
One of the tasks for this sprint is to add functionality to this icon. The idea is to email the user a .csv file of the report.

The current mocks show an export icon, but download was all we could find available.